content/articles/heroku-error-differentapp.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: Troubleshooting Heroku shows different app error
+excerpt: You pointed your domain to Heroku, but when you access your domain you see a different application than the one you expected.
+categories:
+- Heroku and DNSimple
+---
+
+# Troubleshooting Heroku shows different app error
+
+You pointed your domain to Heroku, but when you access your domain from the browser you see a different application than the one you expected.
+
+This error occurs when you point a custom domain to Heroku, but there is another app at Heroku with that domain name set up as a custom domain.

```

Comment:
Adding this as I find it very useful. 

Is there some syntax that I can use to make it clear that it's an example and not a text explanation at the same level of everything else?